libraries/botbuilder-dialogs-adaptive/src/luis/luisAdaptiveRecognizer.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * @module botbuilder-dialogs-adaptive
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Recognizer } from '../recognizers';
+import { StringExpression, ArrayExpression } from 'adaptive-expressions';
+import { LuisPredictionOptions, LuisRecognizerOptionsV3, LuisRecognizer, LuisApplication } from 'botbuilder-ai';
+import { Configurable, DialogContext } from 'botbuilder-dialogs';
+import { BotTelemetryClient, Activity, ActivityTypes } from 'botbuilder-core';
+
+export class LuisAdaptiveRecognizer extends Configurable implements Recognizer {
+    /**
+     * Recognizers unique ID.
+     */
+    public id: string;
+
+    /**
+     * LUIS application ID.
+     */
+    public applicationId: StringExpression;
+
+    /**
+     * LUIS dynamic list.
+     */
+    public dynamicLists: ArrayExpression<any>;
+
+    /**
+     * LUIS endpoint to query.
+     * @remarks
+     * For example: "https://westus.api.cognitive.microsoft.com"
+     */
+    public endpoint: StringExpression;
```

Comment:
Is this the completed endpoint with the AppId? Or is this the host?

What value is [`ExpressionProperty.prototype.getValue`](https://github.com/microsoft/botbuilder-js/blob/1a2a64a7a0fa2cb9da65f9dfdf88d22fa78a8000/libraries/adaptive-expressions/src/expressionProperties/expressionProperty.ts#L57) supposed to return?

Reply:
I copied it verbatim from C# so I believe its just endpoint URL as in the comments.